Mathlib/Algebra/Lie/Weights/Chain.lean

```diff
@@ -212,11 +212,16 @@ lemma exists_forall_mem_corootSpace_smul_add_eq_zero
   have h₃ : genWeightSpaceChain M α χ p q = ⨆ i ∈ Finset.Ioo p q, N i := by
     simp_rw [genWeightSpaceChain_def', LieSubmodule.iSup_coe_toSubmodule]
   rw [← trace_toEnd_genWeightSpaceChain_eq_zero M α χ p q hp hq hx,
-    ← LieSubmodule.toEnd_restrict_eq_toEnd,
-    LinearMap.trace_eq_sum_trace_restrict_of_eq_biSup _ h₁ h₂ (genWeightSpaceChain M α χ p q) h₃]
+    ← LieSubmodule.toEnd_restrict_eq_toEnd]
+  -- This lays bare the horror of trying to treat `LieSubmodule` as both a
```

Comment:
This made me laugh out loud but I think we should probably tone it down a bit:
```suggestion
  -- The lines below illustrate the cost of treating `LieSubmodule` as both a
```
(or something)